src/Illuminate/Validation/Rules/DatabaseRule.php

```diff
@@ -145,10 +146,10 @@ public function whereNotNull($column)
      * Set a "where in" constraint on the query.
      *
      * @param  string  $column
-     * @param  array  $values
+     * @param  \Illuminate\Contracts\Support\Arrayable|array  $values
      * @return $this
      */
-    public function whereIn($column, array $values)
+    public function whereIn($column, $values)
```

Comment:
We can't change a method signature on a minor release since it's potentially breaking to anyone overwriting it.

Reply:
got it ... should i close it and open a new one for 9.x ... or also in 9.x we don't won't to introduce breaking behavior like this ?

Reply:
You can try master yeah